src/database/index.ts

```diff
@@ -51,19 +51,25 @@ const connectionOptions = (() => {
 let pool: Pool;
 let serverReady = false;
 
-setTimeout(() => {
-  pool = createPool({
-    connectTimeout: 60000,
-    trace: false,
-    ...connectionOptions,
-    typeCast,
-  });
+const mysqlInit = () => {
+    pool = createPool({
+      connectTimeout: 60000,
+      trace: false,
+      ...connectionOptions,
+      typeCast,
+    });
 
-  pool.query(mysql_transaction_isolation_level, (err) => {
-    if (err) return console.error(`^3Unable to establish a connection to the database!\n^3[${err}]^0`);
-    console.log(`^2Database server connection established!^0`);
-    serverReady = true;
-  });
-});
+    pool.query(mysql_transaction_isolation_level, (err) => {
+      if (err) {
+        console.error(`^3Unable to establish a connection to the database!\n^3[${err}]^0`);
+        if (mysql_reconnect_time > 0) setTimeout(mysqlInit, mysql_reconnect_time);
+        return;
+      }
+      console.log(`^2Database server connection established!^0`);
+      serverReady = true;
+    });
```

Comment:
Super nitpicky formatting.
```suggestion
  pool.query(mysql_transaction_isolation_level, (err) => {
    if (err) {
      console.error(`^3Unable to establish a connection to the database!\n^3[${err}]^0`);
      if (mysql_reconnect_time > 0) setTimeout(mysqlInit, mysql_reconnect_time);
      return;
    }
    console.log(`^2Database server connection established!^0`);
    serverReady = true;
  });
```